src/helpers/figures/charts/chart_common.ts

```diff
@@ -273,6 +279,49 @@ export function toExcelDataset(getters: CoreGetters, ds: DataSet): ExcelChartDat
     };
   }
 
+  let trend: ExcelChartTrendConfiguration | undefined = undefined;
+  if (ds?.trend?.type) {
+    trend = {
+      type: CHART_TRENDLINE_TYPE_CONVERSION_MAP_REVERSE[ds.trend.type],
+    };
+    if (ds.trend.color) {
+      trend = {
+        ...trend,
+        color: ds.trend.color,
+      };
+    }
+    if (ds.trend.type === "polynomial" && ds.trend.order) {
+      if (ds.trend.order === 1) {
+        trend = {
+          ...trend,
+          type: "linear",
+        };
+      } else {
+        trend = {
+          ...trend,
+          order:
+            ds.trend.order > MAX_XLSX_POLYNOMIAL_DEGREE
+              ? MAX_XLSX_POLYNOMIAL_DEGREE
+              : ds.trend.order,
+        };
+      }
+    }
+    if (ds.trend.type === "trailingMovingAverage" && ds.trend.window) {
+      trend = {
+        ...trend,
+        window: ds.trend.window,
+      };
+    }
+  }
+  if (trend) {
```

Comment:
just add the (possibly undefined) trend to the original return, there's no need to create a new return case with everything the same just to add the trend.

This whole block generating the trendLine can probably be simplified to 10 or so lines of code instead of 40